bokehjs/src/lib/core/kinds.ts

```diff
@@ -637,7 +637,12 @@ export const Color = new Kinds.Color()
 export const Auto = Enum("auto")
 export const CSSLength = new Kinds.CSSLength()
 
-export const FontSize = String
-export const Font = String
-export const Angle = Number
-export const Float = Number
+export const FontSize = Str
+export const Font = Str
+export const Angle = Float
+
+// backwards compatibility aliases
+export const Boolean = Bool
+export const String = Str
+export const Number = Float
+export const Map = Mapping
```

Comment:
Only comment is that these could be under minimal test (e.g. at least for existence) if they are intended to be supported (noting since all the tests were updated to only use the new names)

Reply:
I renamed all conflicting names and added tests. In a later PR I will update the codebase to use the new naming convention and and deprecate these aliases. In the long term this should help with resolving issue #13055.